source/frontend/src/hooks/repositories/mapLayer/usePimsPropertyLayer.tsx

```diff
@@ -58,7 +61,7 @@ export const usePimsPropertyLayer = () => {
       // TODO: Enhance useLayerQuery to allow generics to match the Property types
       const forceCasted = featureCollection as FeatureCollection<
         Geometry,
-        PIMS_Property_Location_View
+        PIMS_Property_Boundary_View
```

Comment:
hmm, is this correct?

Reply:
yes, this was the whole reason for changing it